### PR TITLE
[SAI submodule update] Enable support for SAI v1.11.0

### PR DESCRIPTION
**Pull request by richardyu-ms**
Signed-off-by: richardyu-ms <richard.yu@microsoft.com>

### CODE REVIEW

**Comment by kcudnik** [comment]
looks good, but check build errors

**Review by kcudnik — DISMISSED** [review]

**Comment by prsunny** [comment]
@richardyu-ms , can we merge this?

**Comment by richardyu-ms** [comment]
> @richardyu-ms , can we merge this?

Still cannot merge this PR, cause this change will break the image build for some vendors in sonic-buildimage, and need a correlated upgrade or a period for the acknowledgment.
Already notify some impacted vendors.

File: meta/SaiSerialize.cpp [hunk 1]
```diff
@@ -4087,6 +4262,8 @@ void sai_deserialize_free_attribute_value(
         case SAI_ATTR_VALUE_TYPE_IP_ADDRESS:
         case SAI_ATTR_VALUE_TYPE_IP_PREFIX:
         case SAI_ATTR_VALUE_TYPE_OBJECT_ID:
+        case SAI_ATTR_VALUE_TYPE_JSON:
```
Comment (jimmyzhai) [inline meta/SaiSerialize.cpp:4265]: Need sai_free_list to free it

**Review by jimmyzhai — APPROVED** [review]

**Review by jimmyzhai — DISMISSED** [review]

**Review by jimmyzhai — APPROVED** [review]

**Pull request merged**